docs/changelog.md

```diff
@@ -1,5 +1,42 @@
 ## Changelog
 
+### v2.18.1 (15/11/2021)
+
+#### Highlights
+
+- Release [QueryInst](http://arxiv.org/abs/2105.01928) pre-trained weights (#6460)
+- Support plot confusion matrix (#6344)
+
+#### New Features
+
+- Release [QueryInst](http://arxiv.org/abs/2105.01928) pre-trained weights (#6460)
+- Support plot confusion matrix (#6344)
+
+#### Bug Fixes
+
+- Fix aug test error when the number of prediction bboxes is 0 (#6398)
+- Fix SpatialReductionAttention in PVT (#6488)
+- Fix wrong trunc_normal_init usage in PVT and Swin-Transformer (#6432)
+
+#### Improvements
+
+- Save coco summarize print information to logger (#6505)
```

Comment:
make it a readable sentence: Save the printed AP information of COCO API to logger